examples/widgets/switch/lv_example_switch_3.c

```diff
@@ -0,0 +1,40 @@
+#include "../../lv_examples.h"
+#if LV_USE_SWITCH && LV_BUILD_EXAMPLES
+
+static void event_handler(lv_event_t * e)
+{
+    lv_event_code_t code = lv_event_get_code(e);
+    lv_obj_t * obj = lv_event_get_target(e);
+    if(code == LV_EVENT_VALUE_CHANGED) {
+        LV_UNUSED(obj);
+        LV_LOG_USER("State: %s\n", lv_obj_has_state(obj, LV_STATE_CHECKED) ? "On" : "Off");
+    }
+}
+
+void lv_example_switch_3(void)
+{
+    lv_obj_set_flex_flow(lv_screen_active(), LV_FLEX_FLOW_ROW);
+    lv_obj_set_flex_align(lv_screen_active(), LV_FLEX_ALIGN_CENTER, LV_FLEX_ALIGN_CENTER, LV_FLEX_ALIGN_CENTER);
+
+    lv_obj_t * sw0;
+    lv_style_t* style0;
```

Comment:
```suggestion
    lv_style_t style0;
```

Reply:
This is outdated. The newest commit has this fixed. CI passes now.